lib/components/molecules/StickyHeader/StickyHeader.style.js

```diff
@@ -0,0 +1,10 @@
+import { css } from 'styled-components';
+
+const styles = css`
+  header {
```

Comment:
Have you tested this in all browsers that 
- Sticky Behaviour is consistent. 
- Page not cleating a flicker on content below header component when it becomes sticky.